### PR TITLE
Ot147 62

### PR DESCRIPTION
**Pull request by Lucasfranco1**
**SUMARY**

- Send mail in register
- Send mail in form contact
- Adding var env in applicattion.properties for send grid
- Update EmailService
- Delete template plantilla the email
- Adding template for form contact save

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -9,8 +9,8 @@ spring.jpa.hibernate.ddl-auto: update
 spring.jpa.show-sql:true
 spring.jpa.properties.hibernate.format_sql: false
 #SENDGRID
-app.sendgrid.key:
-app.sendgrid.from-address:
+app.sendgrid.key: ${SENDGRID}
```
Comment (pablofrankenn) [inline src/main/resources/application.properties:12]: Please add those variables to the ReadMe.

Comment (Lucasfranco1) [reply src/main/resources/application.properties:12]: done

File: src/main/java/com/alkemy/ong/auth/service/UserDetailsCustomService.java [hunk 2]
```diff
@@ -36,6 +39,7 @@ public UserEntity save(UserEntity user) {
         userEntity.setEmail(user.getEmail());
         userEntity.setPassword(passwordEncoder.encode(user.getPassword()));
         userEntity.setRoleId(user.getRoleId());
+        emailService.sendWelcomeEmail("blackgrafic30@gmail.com", user.getEmail());
```
Comment (DanteIturri) [inline src/main/java/com/alkemy/ong/auth/service/UserDetailsCustomService.java:42]: put in an environment variable

**Review by DanteIturri — COMMENTED** [review]
change email in an environment variable

File: src/main/resources/templates/confirmationEmail.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xmlns:o="urn:schemas-microsoft-com:office:office">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <meta name="x-apple-disable-message-reformatting">
+  <title>Confirmación de contacto</title>
```
Comment (anavduo) [inline src/main/resources/templates/confirmationEmail.html:8]: May be in English?

**Pull request closed without merging**